packages/ui/src/organisms/ProductCard/ProductCardTitle.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes, AriaAttributes } from 'react'
+
+export interface ProductCardTitleProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  'aria-label'?: AriaAttributes['aria-label']
+}
+
+const ProductCardTitle = forwardRef<HTMLDivElement, ProductCardTitleProps>(
+  function ProductCardTitle(
+    {
+      testId = 'store-product-card-title',
+      'aria-label': ariaLabel = 'Product title',
```

Comment:
I don't think we need this, because we want the screen reader to read the title that you going to pass as `children` and not `Product title`. Maybe what you were trying to do was to apply an `aria-labelledby`? But, not sure where we could add the `id` in this case 🤔

Reply:
@hellofanny that makes sense, I misused the `aria-label` attribute. I don't think `aria-labelledby` would work for this case, so let's keep only with `aria-label` as optional attribute 👍🏼